packages/esm-ward-app/src/ward-patient-card/row-elements/ward-patient-transfer.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useMemo } from 'react';
+import { Movement } from '@carbon/react/icons';
+import styles from '../ward-patient-card.scss';
+import { useTranslation } from 'react-i18next';
+import { type WardPatient } from '../../types';
+
+export interface WardPatientTransferProps {
+  wardPatient: WardPatient;
+}
+
+const WardPatientTransfer: React.FC<WardPatientTransferProps> = ({ wardPatient }) => {
+  const { t } = useTranslation();
+
+  const { dispositionType } = wardPatient?.inpatientRequest;
+  const message = useMemo(() => {
+    if (dispositionType === 'TRANSFER') {
+      return t('transferToDeliveryWard', 'Transfer to Delivery ward');
```

Comment:
Sorry, we weren't clear but instead of being hardcoded to Delivery Ward, this should display wardPatient.inpatientRequest.dispositionLocation.name.

Reply:
If the `dispositionLocation` is null, what should be displayed? cc @mogoodrich

Reply:
Thanks @usamaidrsk !  Let's just display "Pending Transfer" in that case.